NetKAN/TweakScaleRescaled-Redist.netkan

```diff
@@ -0,0 +1,26 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled-Redist
+name: TweakScale Rescaled Redistributable
+abstract: >-
+  Provides a mechanism for other mods to interact with TweakScaleRescaled.
+  This plugin has no overhead when TweakScaleRescaled is not installed.
+$kref: '#/ckan/spacedock/3514'
+$vref: '#/ckan/ksp-avc/GameData/TweakScale/TweakScale-Redist.version'
```

Comment:
This version file is installed by `TweakScaleRescaled`, but it seems like it should be installed by `TweakScaleRescaled-Redist`. I suppose that happened because you put it in `GameData/TweakScale`. Could it go directly in `GameData` instead? Then we could add an install stanza for it here without confusing MM that TS is installed.

Reply:
That makes sense.  It probably doesn't even actually need to get installed right?  Except for people running AVC locally?  I'll move it though (I just hate extra stuff in the root of gamedata :/ )

Reply:
Right, KSP-AVC users are the only ones that would be affected by this.